pyfluidsynth.py

```diff
@@ -624,8 +751,127 @@ def fluid_synth_write_s16_stereo(synth, len):
     fluid_synth_write_s16(synth, len, buf, 0, 2, buf, 1, 2)
     return numpy.frombuffer(buf[:], dtype=numpy.int16)
 
+GEN_STARTADDROFS = 0
+GEN_ENDADDROFS = 1
+GEN_STARTLOOPADDROFS = 2
+GEN_ENDLOOPADDROFS = 3
+GEN_STARTADDRCOARSEOFS = 4
+GEN_MODLFOTOPITCH = 5
+GEN_VIBLFOTOPITCH = 6
+GEN_MODENVTOPITCH = 7
+GEN_FILTERFC = 8
+GEN_FILTERQ = 9
+GEN_MODLFOTOFILTERFC = 10
+GEN_MODENVTOFILTERFC = 11
+GEN_ENDADDRCOARSEOFS = 12
+GEN_MODLFOTOVOL = 13
+GEN_UNUSED1 = 14
+GEN_CHORUSSEND = 15
+GEN_REVERBSEND = 16
+GEN_PAN = 17
+GEN_UNUSED2 = 18
+GEN_UNUSED3 = 19
+GEN_UNUSED4 = 20
+GEN_MODLFODELAY = 21
+GEN_MODLFOFREQ = 22
+GEN_VIBLFODELAY = 23
+GEN_VIBLFOFREQ = 24
+GEN_MODENVDELAY = 25
+GEN_MODENVATTACK = 26
+GEN_MODENVHOLD = 27
+GEN_MODENVDECAY = 28
+GEN_MODENVSUSTAIN = 29
+GEN_MODENVRELEASE = 30
+GEN_KEYTOMODENVHOLD = 31
+GEN_KEYTOMODENVDECAY = 32
+GEN_VOLENVDELAY = 33
+GEN_VOLENVATTACK = 34
+GEN_VOLENVHOLD = 35
+GEN_VOLENVDECAY = 36
+GEN_VOLENVSUSTAIN = 37
+GEN_VOLENVRELEASE = 38
+GEN_KEYTOVOLENVHOLD = 39
+GEN_KEYTOVOLENVDECAY = 40
+GEN_INSTRUMENT = 41
+GEN_RESERVED1 = 42
+GEN_KEYRANGE = 43
+GEN_VELRANGE = 44
+GEN_STARTLOOPADDRCOARSEOFS = 45
+GEN_KEYNUM = 46
+GEN_VELOCITY = 47
+GEN_ATTENUATION = 48
+GEN_RESERVED2 = 49
+GEN_ENDLOOPADDRCOARSEOFS = 20
+GEN_COARSETUNE = 51
+GEN_FINETUNE = 52
+GEN_SAMPLEID = 53
+GEN_SAMPLEMODE = 54
+GEN_RESERVED3 = 55
+GEN_SCALETUNE = 56
+GEN_EXCLUSIVECLASS = 57
+GEN_OVERRIDEROOTKEY = 58
+GEN_PITCH = 59
 
 # Object-oriented interface, simplifies access to functions
+class RamSoundFont:
+    @staticmethod
+    def create_sample ( name : str, wave : Wave_read, rootkey : int ) -> c_void_p:
+        sample = new_fluid_ramsample()
+
+        fluid_sample_set_name( sample, create_string_buffer( name.encode(), 20 ) )
+
+        assert wave.getsampwidth() == 2, f"Samples must be 16bit WAVE files, got {wave.getsampwidth() * 8}bit"
+        assert wave.getnchannels() == 1, f"Samples must be mono WAVE files, got {wave.getnchannels()} channels"
+        assert wave.getframerate() == 44100, f"Samples must be 44100hz WAVE files, got {wave.getframerate()}hz"
+
+        nbframes = wave.getnframes()
+
+        if nbframes == 2147483647:
```

Comment:
Please add a comment about where this number comes from.
```suggestion
        if nbframes == 2_147_483_647:
```